inception-pdf-editor/src/main/js/pdfanno/src/pdfanno.js

```diff
@@ -108,6 +108,24 @@ window.addEventListener('DOMContentLoaded', async () => {
 
 })
 
+// BEGIN INCEpTION EXTENSION - #802 - Rendering annotations pagewise
+function getAnnotations() {
+  var data = {
+    "action": "getAnnotations",
+    "page": window.pageNumber
+  }
+  parent.Wicket.Ajax.ajax({
+    "m": "POST",
+    "ep": data,
+    "u": window.apiUrl,
+    "sh": [],
+    "fh": [function () {
+      console.log('Something went wrong on requesting annotations from inception backend.')
```

Comment:
Wouldn't it be good to show this to the user, e.g. via a JS popup?

Reply:
With `window.alert()`?
Or do we have Wicket, Webanno or some other stuff for this?